components/counter-bar/src/index.js

```diff
@@ -63,166 +61,172 @@ const Counter = styled('button')(
+ *     <CounterBar.Item score={2}>Counter 3</CounterBar.Item>
+ *     <CounterBar.Item score={0}>Counter 4</CounterBar.Item>
+ *     <CounterBar.Item score={2}>Counter 5</CounterBar.Item>
+ *     <CounterBar.Item score={0}>Counter 6</CounterBar.Item>
+ *     <CounterBar.Item score={2}>Counter 7</CounterBar.Item>
+ *   </CounterBar.Container>
+ * </CounterBar>
  * ```
- * 
  * Zero/no scores
- * ```jsx 
- * <CounterBar
- *  listTitle="All counters"
- *  name="name"
- *  counters={[
- *    { id: 'abc', name: 'Counter 1', score: 0 },
- *    { id: 'def', name: 'Counter 2' },
- *    { id: 'ghi', name: 'Counter 3', score: 0 },
- *  ]}
- * />
+ * ```jsx
+ * <CounterBar>
+ *   <CounterBar.Total score={0}>All counters</CounterBar.Total>
+ *   <CounterBar.Container>
+ *     <CounterBar.Item score={0}>Counter 1</CounterBar.Item>
+ *     <CounterBar.Item>Counter 2</CounterBar.Item>
+ *     <CounterBar.Item score={0}>Counter 3</CounterBar.Item>
+ *   </CounterBar.Container>
+ * </CounterBar>,
  * ```
  */
-const CounterBar = ({ activeTitle, listTitle, counters, onSelect, ...props }) => {
-  const totalScore = counters.reduce((a, b) => a + (Number(b.score) || 0), 0);
+const CounterBar = ({ children, ...props }) => {
   return (
-    <Container {...props}>
-      <CountWrapper active={activeTitle} disabled={!totalScore} onClick={() => onSelect()}>
-        <ResultCountTitle
-          count={totalScore}
-          countColor={totalScore > 0 ? WHITE : BLACK}
-          countBackgroundColor={totalScore > 0 ? RED : GREY_3}
-        >
-          {listTitle}
-        </ResultCountTitle>
-      </CountWrapper>
-      {counters.map(counter => (
-        <Counter
-          active={counter.active}
-          disabled={!counter.score}
-          key={counter.id}
-          name={counter.id}
-          onClick={() => onSelect(counter.id)}
-          empty={!counter.name || counter.name.length === 0}
-        >
-          <ResultCountTitle
-            countColor={counter.score > 0 ? WHITE : BLACK}
-            countBackgroundColor={counter.score > 0 ? RED : GREY_3}
-            count={counter.score || 0}
-          >
-            {counter.name}
-          </ResultCountTitle>
-        </Counter>
-      ))}
-    </Container>
+    <Wrapper {...props}>
+      {children}
+    </Wrapper>
   );
 };
 
 CounterBar.propTypes = {
-  activeTitle: PropTypes.bool,
-  counters: PropTypes.arrayOf(
-    PropTypes.shape({
-      active: PropTypes.bool,
-      id: PropTypes.any.isRequired,
-      name: PropTypes.string,
-      score: PropTypes.number,
-    }),
-  ),
-  listTitle: PropTypes.any,
-  name: PropTypes.string.isRequired,
+  children: PropTypes.node.isRequired,
+};
+
+CounterBar.Total = ({active, children, onSelect, score}) => {
+  return (
+    <TotalWrapper active={active} disabled={!score}>
+      <ResultCountTitle
+        count={score}
+        countColor={score > 0 ? WHITE : BLACK}
+        countBackgroundColor={score > 0 ? RED : GREY_3}
```

Comment:
this needs to be configurable - so the `Total` should take `prop` values for `scoreBackgroundColor` and `scoreDisabledBackgroundColor` - default those values to `RED` and `GREY_3` respectively

(arguably one could not bother with the `Disabled` variants)

same comment applies to `countColor` too

Reply:
Right, I have added all of these but can rearrange in any way you like.